build/.vsts-ci.yml

```diff
@@ -51,7 +51,7 @@ jobs:
 - job: raspberrypi
   variables:
     _PREVIEW_VSTS_DOCKER_IMAGE: "aziotbld/raspberrypi-c-buster:brown"
-  pool: aziotbld-lin01
+  pool: Hosted Ubuntu 1604
```

Comment:
just out of curiosity why aren't we bumping to 1804? Aren't we on a two version sliding window so now on 1804 and 2004 support?

Reply:
I'm going to move this to our 1ES hosted agents.

Reply:
BTW, this is the OS version of the hosted agent, not the container running the build...yes that is still 1604. We need to update our containers, which is out of scope for this.